README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
+| `appName`                                  | The name of the application as shown in the Unleash UI. Registered applications are listed on the Applications page.                                                | Yes      | `null`                               |
+| `backupFile`                               | The path to the file where [local backups](#local-backup) get stored.                                                                                               | No       | `"java.io.tmpdir/unleash-repo.json"` |
+| `customHttpHeader`                         | Add a [custom HTTP header](#custom-http-headers) to the list of HTTP headers that will the client sends to the Unleash API. Each method call will add a new header. | No       | N/A                                  |
+| `customHttpHeadersProvider`                | Add a custom HTTP header provider. Useful for [dynamic custom HTTP headers](#dynamic-custom-http-headers).                                                          | No       | `null`                               |
+| `disableMetrics`                           | A boolean indicating whether the client should disable sending usage metrics to the Unleash server.                                                                 | No       | `false`                              |
+| `enableProxyAuthenticationByJvmProperties` | Enable support for [using JVM properties for HTTP proxy authentication](#http-proxy-with-authentication).                                                           | No       | `false`                              |
+| `environment`                              | The name of the current environment.                                                                                                                                | No       | `null`                               |
+| `fallbackStrategy`                         | ???                                                                                                                                                                 | No       | `null`                               |
+| `fetchTogglesInterval`                     | How often (in seconds) the client should check for toggle updates.                                                                                                  | No       | `10`                                 |
+| `instanceId`                               | An identifier for your instance. Typically a hostname, pod id or something similar. Unleash uses this internally for ...something                                   | Yes      | `null`                               |
+| `namePrefix`                               | ???                                                                                                                                                                 | No       |                                      |
```

Comment:
What is a name prefix? Is it a prefix applied to all toggles?

Reply:
This nudges the SDK to only return toggles that start with a given prefix so if you set this to:

'coolFeature'

And your toggles server side are called:

'notAcoolFeature'
'coolFeature42'

You'll only get the second one back in the SDK call. This supported on the Unleash server but only the Node and Java SDKs make use of it (insofar as I'm aware and confirmed by our compat table)